catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/CswEndpoint.java

```diff
@@ -650,10 +650,12 @@ private InsertResultType getInsertResultFromResponse(CreateResponse createRespon
     private int deleteRecords(DeleteAction deleteAction)
             throws CswException, FederationException, IngestException, SourceUnavailableException,
             UnsupportedQueryException {
-        List<QName> qNames = typeStringToQNames(deleteAction.getTypeName(),
-                deleteAction.getPrefixToUriMappings());
 
-        QueryRequest queryRequest = queryFactory.getQuery(deleteAction.getConstraint(), qNames);
+        QueryRequest queryRequest = queryFactory.getQuery(deleteAction.getConstraint());
+        if (!CswConstants.CSW_RECORD.equals(deleteAction.getTypeName())) {
```

Comment:
Why are the tags only updated if they typeName is something other than `csw:Record`?